.github/workflows/publish-helm-chart.yml

```
@@ -0,0 +1,170 @@
+        with:
+          version: '3.14.0'
+
+      - name: Configure AWS credentials (OIDC)
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.AWS_ROLE_ARN }}
+          aws-region: ${{ env.AWS_REGION }}
+        continue-on-error: true
+
+      - name: Configure AWS credentials (Fallback - Static Credentials)
+        if: ${{ !secrets.AWS_ROLE_ARN }}
```

actionlint (1.7.8)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)